error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestInput.java

```diff
@@ -422,6 +422,18 @@ ImmutableSet<StepVerifier.Step<String>> testStepVerifierStepExpectNext() {
         StepVerifier.create(Mono.just("baz")).expectNextMatches("qux"::equals));
   }
 
+  ImmutableSet<StepVerifier.Step<?>> testFluxAsStepVerifierExpectNext() {
+    return ImmutableSet.of(
+        Flux.just(1)
+            .collect(toImmutableList())
+            .as(StepVerifier::create)
+            .assertNext(list -> assertThat(list).containsExactly(2)),
+        Flux.just(3)
+            .collect(toCollection(ArrayList::new))
+            .as(StepVerifier::create)
+            .assertNext(list -> assertThat(list).containsExactly(4)));
+  }
```

Comment:
After pushing this I realized that perhaps for rule one test case would suffice, or at least be more consistent with what we do elsewhere. :shrug:

Reply:
I agree, I'll add a commit to drop the second test case.